optuna/integration/tensorflow.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import
+
+import optuna
+
+
+try:
+    import tensorflow as tf
+    from tensorflow.train import SessionRunHook
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # PruningHook is disabled because TensorFlow is not available.
+    _available = False
+    SessionRunHook = object
+
+
+class TensorFlowPruningHook(SessionRunHook):
+    """TensorFlow SessionRunHook to prune umpromising trials.
+
+    Example:
+
+        Add a pruning SessionRunHook for a TensorFlow's Estimator.
+
+        .. code::
+
+                optuna_pruning_hook = OptunaPruningHook(
+                    trial=trial,
+                    estimator=clf,
+                    metric="accuracy",
+                    is_higher_better=True,
+                    run_every_steps=10,
+                )
+                tf.estimator.train_and_evaluate(
+                    clf,
+                    tf.estimator.TrainSpec(input_fn=train_input_fn, max_steps=500, hooks=hooks),
+                    eval_spec
+                )
+    Args:
+        trial:
+            A :class:`~optuna.trial.Trial` corresponding to the current evaluation of
+            the objective function.
+        estimator:
+            A :estimator which you will use.
```

Comment:
I think 'A :estimator' is a typo for 'An estimator'.

Reply:
I fixed it.